dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+
+# "Error creating order for ACME server."
+CWPKI2011E=CWPKI2011E: Failed to create the certificate order for the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2011E.explanation=Creating the certificate order failed and a signed certificate cannot be created.
+CWPKI2011E.useraction=Review the provided exception for details on the failure.
+
+# "Error signing the certificate signing request."
+CWPKI2012E=CWPKI2012E: Failed to sign the certificate signing request for the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2012E.explanation=The certificate order was created, but signing the certificate signing request failed.
+CWPKI2012E.useraction=Review the provided exception for details on the failure.
+
+# "Error ordering the certificate.
+CWPKI2013E=CWPKI2013E: The certificate signing request was created and signed, but the order failed from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2013E.explanation=The certificate order was created and signed, but ordering the certificate failed.
+CWPKI2013E.useraction=Review the provided exception for details on the failure.
+
+#"Error getting the encoded certificate signing request."
+CWPKI2014E=CWPKI2014E: The certificate signing request for the ACME certificate authority at URI {0} was created and signed, but encoding the request failed. The error is {1}.
+CWPKI2014E.explanation=Encoding the certificate signing request failed and a signed certificate cannot be created.
+CWPKI2014E.useraction=Review the provided exception for details on the failure.
+
+# "Error updating the order status from the ACME server."
+CWPKI2015E=CWPKI2015E: Received an exception while requesting a certificate order update from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2015E.explanation=A certificate order was send to the certificate authority. While requesting an status update, an exception occurred.
+CWPKI2015E.useraction=Review the provided exception for details on the failure.
+
+# "Error requesting an existing account from ACME server."
+CWPKI2016E=CWPKI2016E: Failed to request an existing account from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2016E.explanation=Requested an existing account, but received an exception.
+CWPKI2016E.useraction=Review the certificate authority URI provided. Verify that the URI can be successfully accessed by the calling server and the server can receive a callback.
+
+# "Error retrieving the terms of service from the ACME server."
+CWPKI2017E=CWPKI2017E: Failed to receive the terms of service from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2017E.explanation=Requested the terms of service from the certificate authority, but an exception occurred.
+CWPKI2017E.useraction=Review the provided exception for details on the failure. Visit the ACME certificate authority website directly to review the terms of service.
```

Comment:
Visit the ACME certificate authority website to review the terms of service.

(don't need "directly")